src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -575,34 +601,58 @@ public static void closeAllEditorTabs(RemoteRobot remoteRobot) {
         }
     }
 
+    /**
+     * Closes a source file that is open in the editor pane
+     *
+     * @param remoteRobot The RemoteRobot instance.
+     * @param fileName The string file name
+     */
+
+    public static void clickOnFileTab(RemoteRobot remoteRobot, String fileName) {
```

Comment:
recommend to include "CloseFile" in the name ie. `closeFileWithFileTab` or `clickOnFileTabToClose` 
....unless the method header description is not the correct one

Reply:
updated the header description to accurately reflect what the method is now doing